federation-2/router-bridge/js-src/runtime.js

```diff
@@ -17,6 +17,18 @@ function done(result) {
   Deno.core.ops.deno_result(result);
 }
 
+crypto = {
+  getRandomValues: (arg) => {
+    Deno.core.ops.op_crypto_get_random_values(arg);
+    return arg;
+  },
+};
+
+// This comes in handy when debugging dependencies in the node modules.
+console = {
+  log: (...args) => print(...args),
+};
+
```

Comment:
Can you speak more to this?  Why not just debugging using `print`?

My appreciation for using `print` — rather than `console.log` — is that some rogue module that starts printing things with `console` will **just fail** rather than _start logging things to the Router's console_

Reply:
That's fair, removing it.

Reply:
fixed in 573b40c